src/components/oculus-touch-controls.js

```diff
@@ -150,6 +150,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       controller: 0,
       rotationOffset: data.rotationOffset !== -999 ? data.rotationOffset : isRightHand ? -90 : 90
     });
+    el.emit('controller-connected', {type: 'oculus-touch-controller'});
```

Comment:
I think the idea was to avoid the ambiguity of having to know correspondence from gamepad.id to controller component.  Another approach is to listen for `componentinitialized` of `tracked-controls`, and look at `.controllerPresent` on the controller components -- but then not sure how to handle emulation (separate discussion)